src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -333,14 +320,13 @@ You've probably experienced layout shifts due to UI that pops-in at the top or b
     </figcaption>
   </figure>
 
-If you need to display these types of UI affordances, reserve sufficient space in the viewport for it in advance (for example, using a placeholder or skeleton UI) so that when it loads, it does not cause content in the page to surprisingly shift around.
+If you need to display these types of UI affordances, reserve sufficient space in the viewport for it in advance (for example, using a placeholder or skeleton UI) so that when it loads, it does not cause content in the page to surprisingly shift around. Alternatively, ensure the element is not part of the document flow by overlaying the content where this makes sense. See our [Best practices for cookie notices](/cookie-notice-best-practices/) post for more recommendations on these types of components.
```

Comment:
```suggestion
If you need to display these types of UI affordances, reserve sufficient space in the viewport for it in advance (for example, using a placeholder or skeleton UI) so that when it loads, it does not cause content in the page to surprisingly shift around. Alternatively, ensure the element is not part of the document flow by overlaying the content where this makes sense. See the [Best practices for cookie notices](/cookie-notice-best-practices/) post for more recommendations on these types of components.
```

Reply:
Style guide says to avoid first person.